core/src/main/java/org/apache/carbondata/core/writer/CarbonDeleteDeltaWriterImpl.java

```diff
@@ -81,7 +80,9 @@ public CarbonDeleteDeltaWriterImpl(String filePath, FileFactory.FileType fileTyp
       if (null != dataOutStream) {
         dataOutStream.flush();
       }
-      CarbonUtil.closeStreams(brWriter, dataOutStream);
+      if (null != brWriter) {
+        brWriter.close();
+      }
```

Comment:
there is a catch block above in this method for IOException.....there we are only logging the exception...kindly add a throw statement in that catch block also after logging the error

Reply:
Fixed